src/js/index.js

```diff
@@ -248,16 +248,58 @@ export const hpe = deepFreeze({
     },
   },
   checkBox: {
-    color: 'selected-text',
-    gap: 'small',
-    check: {
-      radius: '2px',
+    hover: {
+      border: {
+        color: undefined,
+      },
+      background: {
+        color: 'background-contrast',
+      },
     },
     border: {
+      color: 'border',
       width: '1px',
     },
+    check: {
+      radius: '2px',
+      extend: ({ theme, checked, indeterminate }) => `
+      background-color: ${
+        checked || indeterminate
+          ? theme.global.colors['green!']
+          : theme.global.colors.background[theme.dark ? 'dark' : 'light']
+      };
+      ${(checked || indeterminate) && 'border: none;'}
+        `,
+    },
+    icon: {
+      size: '18px',
+      extend: 'stroke: white;',
+    },
+    gap: 'small',
+    toggle: {
+      background: 'background',
+      color: 'background',
+      knob: {
+        extend: ({ theme }) => `
+           box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.12);
```

Comment:
Trying to tie styling back to globals where appropriate:

```suggestion
           box-shadow: ${
              theme.global.elevation[theme.dark ? 'dark' : 'light'].small
            };
```